hazelcast/src/main/java/com/hazelcast/jet/Jet.java

```diff
@@ -25,7 +26,8 @@
 /**
  * Entry point to the Jet product.
  *
- * @since 3.0
```

Comment:
Can you keep both `@since` and `@deprecated`?

Reply:
Added `@since` tags again.